backend/ethereum/channel/test/simulated.go

```diff
@@ -142,3 +148,67 @@ func (s *SimulatedBackend) StartMining(interval time.Duration) {
 func (s *SimulatedBackend) StopMining() {
 	close(s.mining)
 }
+
+// Reorg simulates a chain reorg with parameters depth and length.
```

Comment:
The terms `depth` and `length` are common when talking about reorgs in ETH, i will still document them.  

Maybe this info graphic from whimsical helps you understand. 
![reorg](https://user-images.githubusercontent.com/10380170/129357188-f14b48c1-ace2-4063-9af8-3a378099984b.png)

Reply:
PS: Regarding your question:  
It could be done, but i think the usage is more compact when `length` is explicitly specified.  
In the testing code i often use `txs` which has length 2 even when the reorg `lenght` parameter is set to 3.  
This then implicitly adds `length - size(reorder(txs))` empty blocks to the end of the chain.

Reply:
Yes looks good, is also a bit more compact.  
I added you as "co-authored-by".